### PR TITLE
Fixed notification bug

### LINKED ISSUE #62: Fix update & notification logic
There's a bug in the auto updates. 

Steps to repeat: 
 - open pulse and dev console
 - add a new asset via form
 - wait for check. console reports `updated:  false` 
 - new project doesn't appear in the stream
 - notification isn't triggered

Note: we should fix this, but it is lower priority than other issues in this milestone.

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #62.

PR https://github.com/mozilla/network-pulse/pull/124 turns off auto refresh & updates (for now) while this PR fixes the currently broken notification logic

To test

- make sure `FEATURE.notify` in `pulse.js` is set to `true`
- you can lower `REFRESH_INTERVAL` if you are like me who's impatient 😉 
- try submitting a new project, return to the app, and see you get a desktop notification for that newly added project

### CODE REVIEW

File: js/notifier.js [hunk 1]
```diff
@@ -49,13 +49,18 @@ var Notifier = {
     var updated = false;
     var newestProject = sortedProjects[0];
     var newestTimestamp = Date.parse(newestProject.Timestamp);
-    var lastNewestProjectTimestamp = Number(localStorage.getItem("newestProjectTimestamp"));
-    if (lastNewestProjectTimestamp && lastNewestProjectTimestamp < newestTimestamp) {
```
Comment (mmmavis) [inline js/notifier.js]: fixed a bug here - we forgot to check for the initial load and therefore `newestProjectTimestamp` never got saved to `localStorage` (i.e., this `if` condition we had before could never be met)

File: js/pulse.js [hunk 2]
```diff
@@ -106,7 +106,6 @@ var newProjectForm = {
     newProjectForm.formContainer.style.display = 'none';
     newProjectForm.projectContainer.style.display = 'block';
     newProjectForm.toggleFormButton.style.transform = 'rotate(0deg)';
-    setTimeout(function(){ PulseMaker.getData(); }, 250);
```
Comment (mmmavis) [inline js/pulse.js]: no need to do this again since `refresh` is already a recurring task
(see `pulse.js`, the `PulseMaker.init` method)

**Comment by mmmavis** [comment]
@xmatthewx @cadecairos @acabunoc review? 🍰 🍕 🍹

File: js/pulse.js (hunk 2 above)
Comment (acabunoc) [inline js/pulse.js]: Can we update the comment here
>  this turns on both auto app refresh and browser notification

 https://github.com/mmmavis/network-pulse/blob/9c2ed73f25c92a0a2d10d281d617d35bbe07f465/js/pulse.js#L7

To only mention browser notifications?

Comment (acabunoc) [inline js/pulse.js]: Nevermind, just noticed the refresh is still wrapped in `if(FEATURE.notify)` :)

**Comment by acabunoc** [comment]
So I can see the "updated: true" but no desktop notification. I granted permissions & ran this on a local server, so I'm not sure what I'm missing... @mmmavis ?

File: js/pulse.js (hunk 2 above)
Comment (mmmavis) [inline js/pulse.js]: I will clean up the refresh & notification logic after we have decided how it should work 😉 We have a few tickets open for this discussion!

**Comment by mmmavis** [comment]
Just tried it again on FF and it was working as expected to me. Not sure what's causing the problem... but since the notification feature is currently turned off, @acabunoc and I are good with merging this PR now and revisit this issue when we decide to turn the feature back on.

**Comment by mmmavis** [comment]
🍸 🍸 🍸 🍸

**Pull request merged**